devel/mdds/Portfile

```diff
@@ -4,8 +4,9 @@ PortSystem          1.0
 PortGroup           gitlab 1.0
 PortGroup           boost 1.0
 
-gitlab.setup        mdds mdds 2.0.0
-revision            0
+gitlab.setup        mdds mdds 1.7.0
+revision            1
```

Comment:
no need to increase the `revision` as well

Reply:
That is there because of this commit (the one prior to bumping to 2.0.0): https://github.com/macports/macports-ports/commit/3274c3a7776d9058fa9bf36ebd4925d4d0d9bef2#diff-c521366ba7481be728c8078d630132b64abd4f335786436b3e9a79bb0f105009

Reply:
oh, I see - sorry my bad, that wasn't clear from this diff...